src/main/java/orm/TableEntity.java

```diff
@@ -88,15 +136,16 @@ private TablePrimaryField extractId(Class<ENTITY> entityClass) {
             throw new InvalidIdMappingException("Entity must have one @Id field");
         }
 
-        return new TablePrimaryField(idList.getFirst(), jpaSettings);
+        return new TablePrimaryField(idList.getFirst(), entity, jpaSettings);
     }
 
     /**
      * 모든 영속성 필드 추출
+     *
      * @param entityClass
      * @return
```

Comment:
```suggestion
     * @return List<TableField>
```
void가 아닌데 빠진것 같아요! 😂

Reply:
앗 그렇네요 이부분 수정하도록 하겠습니다 :)